doc/source/serve/key-concepts.rst

```diff
@@ -54,14 +55,15 @@ It's important to note that Ray Serve places these backends in individual worker
     return "hello world"
 
   class RequestHandler:
-    def __init__(self):
-        self.msg = "hello, world!"
+    def __init__(self, msg):
+        self.msg = msg
 
     def __call__(self, flask_request):
         return self.msg
 
   serve.create_backend("simple_backend", handle_request)
-  serve.create_backend("simple_backend_class", RequestHandler)
+  # We pass in the message that the backend will return as an argument.
+  serve.create_backend("simple_backend_class", RequestHandler, "hello, world!")
```

Comment:
can we clarify that when we call this backend we will see "hello, world!"

My worry is that given this, it's not immediately clear that "msg" will be equal to "hello, world!" in this code block. Can we make that more obvious?

Reply:
Tried making it more obvious